database: adicionar tutorial pra criação do banco

ekisu opened the pull request.

ekisu committed:
database: adicionar tutorial pra criação do banco

LucasPCBranco commented on DATABASE_SETUP.md:
```diff
@@ -0,0 +1,47 @@
+# Configurando o banco PostgreSQL
```
Está faltando a parte de definir o caminho do DB, que foi a primeira coisa que a gente descobriu para rodar pelo Heroku com as credenciais locais. No caso, `export DATABASE_URL=postgres://usuario:senha@localhost`

ekisu replied:
Isto seria necessário somente para configurações personalizadas, adicionei uma nota sobre isso.

kex1998 commented on DATABASE_SETUP.md:
````diff
@@ -0,0 +1,47 @@
+$ sudo apt-get install postgresql postgresql-contrib
+```
+
+Em seguida, será necessário alterar o arquivo `/etc/postgresql/10/main/pg_hba.conf`. 
````
dependendo da versão do postgres.
utilizar o comando
```sh
psql --version
```
e trocar o valor de `/etc/postgresql/VERSAO/main/pg_hba.conf`

ekisu replied:
:+1:

ekisu committed:
database: adicionar parte sobre DATABASE_URL

ekisu committed:
database: adicionar nota sobre versão do postgres

ekisu committed:
database: utilizar method trust p/alterar senha

Algumas instalacoes nao permitem que facamos o login no psql sem senha
utilizando o method md5, portanto para criar a primeira senha utilizamos
o metodo trust.
